spec/helpers/state_file/nj_tenant_eligibility_helper_spec.rb

```diff
@@ -111,8 +95,8 @@
                  tenant_shared_rent_not_spouse: "yes", # worksheet required
                  tenant_more_than_one_main_home_in_nj: "no" # advance
         }
-        it "determines the taxpayer should fill out the worksheet" do
-          expect(described_class.determine_eligibility(intake)).to eq(described_class::WORKSHEET)
+        it "determines the taxpayer should advance" do
+          expect(described_class.determine_eligibility(intake)).to eq(described_class::ADVANCE)
```

Comment:
[pebble] Similar to my comment above, this test should be reviewed for what value it is providing us rather than just change the words. The test context "taxpayer selects checkboxes that correspond to both worksheet required and advance states" is no longer possible, so I'd argue this should probably be deleted